ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/tunnel-interfaces.j2

```diff
@@ -0,0 +1,88 @@
+{% if tunnel_interfaces is defined and tunnel_interfaces is not none %}
+
+## Tunnel Interfaces
+
+### Tunnel Interfaces Summary
+
+| Interface | Description | VRF | MTU | Shutdown |
+| --------- | ----------- | --- | ---- | -------- |
+{%     for tunnel_interface in tunnel_interfaces | arista.avd.natural_sort('name') %}
+{%         set description = tunnel_interface.description | arista.avd.default('-') %}
+{%         set vrf = tunnel_interface.vrf | arista.avd.default('default') %}
+{%         set mtu = tunnel_interface.mtu | arista.avd.default('-') %}
+{%         if tunnel_interface.shutdown is arista.avd.defined %}
```

Comment:
just use default filter like all the others.

Reply:
This comes from `documentation/vlan-interfaces.j2` should I clean it as well?

Reply:
Addressed in 0db9c4a